petclinic-frontend/src/features/bills/AdminBillsListTable.tsx

```diff
@@ -315,7 +315,7 @@ export default function AdminBillsListTable(): JSX.Element {
           </p>
         </div>
       ) : (
-        <div className="container">
+        <div className="admin-bills-list-table-container">
```

Comment:
Good job changing it to a bill-specific class so it doesn't mess with other container class such as the navbar :)

Reply:
thank you for catching that for us!